src/api/ledger/parse/trustline.js

```diff
@@ -15,8 +15,10 @@ function parseFlag(flagsValue, trueValue, falseValue) {
   return undefined;
 }
 
-function parseQuality(quality) {
-  return (new BigNumber(quality)).shift(-9).toNumber();
+function parseQuality(quality?: number) {
+  if (typeof quality === 'number') {
```

Comment:
We don't need typechecking here because flow should already be checking that. Can just check `quality !== undefined`. Also, the function should explicitly return `undefined` otherwise.